package.json

```diff
@@ -38,6 +38,16 @@
 	},
 	"contributes": {
 		"commands": [
+			{
+				"command": "intuita.signIn",
+				"title": "Sign in",
+				"icon": "$(sign-in)"
+			},
+			{
+				"command": "intuita.signedIn",
```

Comment:
can you explain the need for this command, from the name it sounds like a state fact

Reply:
This is attached to a user icon that will appear on the right end of the screenshot attached only if user is already signed-in.

`command` is a required field. When pressing this, we will see a VSCode popup saying "You are signed in." with a button that says "Do you want to sign out?"

![Screenshot 2023-09-18 at 10 26 50](https://github.com/intuita-inc/intuita-vscode-extension/assets/32841130/c2200a33-3eb1-4094-bf8c-5a090c92ee7c)